docs/source/index.rst

```diff
@@ -8,7 +8,7 @@ to SOTA neural networks, models combination methods and smart backtesting.
 ETNA is designed to make working with time series simple, productive, and fun.
 
 ETNA is the first python open source framework of
-`Tinkoff.ru <https://www.tinkoff.ru/eng/>`_ Artificial Intelligence Center.
+`Tinkoff.ru <https://www.tbank.ru/>`_ Artificial Intelligence Center.
```

Comment:
Probably, Tinkoff.ru here should also be renamed.

Reply:
Fixed